LICENSE.md

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2016
+Copyright (c) 2017
```

Comment:
Ya, we have fixed the blueprint to not touch this file in future updates.

@kratiahuja - Can you tweak to make it `2015-2017`?